### PR TITLE
Diana/express server

### PR DESCRIPTION
**Pull request by ruslanovna**
merge my MondoDB new feature

### CODE REVIEW

File: server/models/studentModels.js [hunk 1]
```diff
@@ -1 +1,33 @@
...
+// create a collection instance of the model
+const Student = mongoose.model('students', studentSchema);
+
+
+// create a goal schema
+const goalsSchema = new Schema({
+  description: {type: String, required: true}, 
+  dueDate: Number,
+  //behaves like a foreign key referencing the 'students' collection
+  status: {
```
Comment (rsa0810) [inline server/models/studentModels.js:23]: change status to student

File: server/models/studentModels.js [hunk 2]
```diff
@@ -1 +1,33 @@
...
+});
+
+// create a collection instance of the model
+const Student = mongoose.model('students', studentSchema);
+
+
+// create a goal schema
+const goalsSchema = new Schema({
+  description: {type: String, required: true}, 
+  dueDate: Number,
```
Comment (rsa0810) [inline server/models/studentModels.js:21]: change dueDate value to be (instance of) Date

**Review by rsa0810 — CHANGES_REQUESTED** [review]
Please review given comments and make appropriate changes. Thank you!

File: server/models/studentModels.js (hunk 1 above)
Comment (ruslanovna) [reply server/models/studentModels.js:23]: done

File: server/models/studentModels.js (hunk 2 above)
Comment (ruslanovna) [reply server/models/studentModels.js:21]: done

**Pull request closed without merging**